src/ParametricOptInterface.jl

```diff
@@ -201,7 +318,249 @@ function MOI.get(model::ParametricOptimizer, attr::T) where {
+        quad_aff_vars = MOI.ScalarQuadraticTerm{T}[] #outside declaration so it has default value; parameter as variable_index_1
+
+        if (any(haskey(model.parameters, f.quadratic_terms[i].variable_index_1) for i = 1:length(f.quadratic_terms)) ||
+             any(haskey(model.parameters, f.quadratic_terms[i].variable_index_2) for i = 1:length(f.quadratic_terms)))
+            
+            quad_terms = MOI.ScalarQuadraticTerm{T}[]  
+
+            for i in f.quadratic_terms
+                if haskey(model.variables, i.variable_index_1) && haskey(model.variables, i.variable_index_2)
+                    push!(quad_terms, i) # if there are only variables, it remains a quadratic term
+
+                elseif haskey(model.parameters, i.variable_index_1) && haskey(model.variables, i.variable_index_2)
+                    # This is the case when i.variable_index_1 is a parameter and i.variable_index_2 is a variable.
+                    # Thus, it creates an affine term. Convention: param as 1, var as 2
+                    aux = MOI.ScalarQuadraticTerm(i.coefficient, i.variable_index_1, i.variable_index_2)
+                    push!(quad_aff_vars, aux)  
+
+                elseif haskey(model.variables, i.variable_index_1) && haskey(model.parameters, i.variable_index_2)
+                    # Check convention defined above
+                    aux = MOI.ScalarQuadraticTerm(i.coefficient, i.variable_index_2, i.variable_index_1)
+                    push!(quad_aff_vars, aux)          
+
+                elseif haskey(model.parameters, i.variable_index_1) && haskey(model.parameters, i.variable_index_2)
+                    # This is the case where both variable_index_1,2 are actually parameters
+                    aux = MOI.ScalarQuadraticTerm(i.coefficient, i.variable_index_1, i.variable_index_2)
+                    push!(quad_params, aux)
+
+                else
+                    error("Constraint uses a variable that is not in the model")
+                end
+            end
+
+        else
+            quad_terms = f.quadratic_terms
+        end
+
+        aff_terms = MOI.ScalarAffineTerm{T}[]
+
+        for i in quad_aff_vars
+            aux = MOI.ScalarAffineTerm(i.coefficient * model.parameters[i.variable_index_1], i.variable_index_2)
+            push!(aff_terms, aux)
+        end
+
+        aff_terms = vcat(aff_terms, aff_vars)
+
+        const_term = f.constant
+
+        for j in aff_params
+            const_term += j.coefficient * model.parameters[j.variable_index]
+        end
+
+        for j in quad_params
+            const_term += j.coefficient * model.parameters[j.variable_index_1] * model.parameters[j.variable_index_2]
+        end
+
+
+        f_quad = MOI.ScalarQuadraticFunction(
+                    aff_terms,
+                    quad_terms,
+                    const_term 
+                )
+
+        co = MOI.set(model.optimizer, attr, f_quad)
```

Comment:
What do you expect is returned here ? `MOI.set` should return `nothing` even if some optimizer return something sometimes, we cannot rely on that

Reply:
Didn't know that, I tested with GLPk at first and it returned something, wrongfully assumed that it was the standard! But I'm correcting it right now

Reply:
We should probably check that it returns `nothing` in the MOi tests to ensure a consistent behavior